dev/tests/js/jasmine/tests/app/code/Magento/Checkout/frontend/js/view/form/element/email.test.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+/* eslint max-nested-callbacks: 0 */
+
+define(['squire', 'ko'], function (Squire, ko) {
+    'use strict';
+
+    describe('Magento_Checkout/js/view/form/element/email', function () {
+        var injector = new Squire(),
+        mocks = {
+            'Magento_Customer/js/model/customer': {
+                isLoggedIn: ko.observable()
+            },
+            'Magento_Customer/js/action/check-email-availability': jasmine.createSpy(),
+            'Magento_Customer/js/action/login': jasmine.createSpy(),
+            'Magento_Checkout/js/model/quote': {
+                isVirtual: jasmine.createSpy(),
+                shippingAddress: jasmine.createSpy()
+            },
+            'Magento_Checkout/js/checkout-data': jasmine.createSpyObj(
+                'checkoutData',
+                [
+                    'setInputFieldEmailValue',
+                    'setValidatedEmailValue',
+                    'setCheckedEmailValue',
+                    'getInputFieldEmailValue',
+                    'getValidatedEmailValue',
+                    'getCheckedEmailValue'
+                ]
+            ),
+            'Magento_Checkout/js/model/full-screen-loader': jasmine.createSpy(),
+            'mage/validation': jasmine.createSpy()
+        },
+        obj;
+
+        beforeEach(function (done) {
+            window.checkoutConfig = {
+                quoteData: {
+                    /* jscs:disable requireCamelCaseOrUpperCaseIdentifiers */
+                    entity_Id: 1
```

Comment:
I think it should be like this:
```suggestion
                    entity_id: 1
```
Is it mistake?

Reply:
https://github.com/magento/magento2/blob/2.4-develop/dev/tests/js/jasmine/tests/app/code/Magento/Vault/view/frontend/web/js/view/payment/method-renderer/vault.test.js#L48
This file still use entity_Id

Reply:
But i think use entity_id is more make sense than entity_Id

Reply:
Could you check what we have on the frontend? If it's entity_Id - let's leave it test as is, if it's entity_id - let's update the test